### PR TITLE
Decoding Labels 13 through 18 preamble '/'

### PR DESCRIPTION
**Pull request by makrsmark**
* label 13-16 are OOOI events
* Label 17 and 18 are reports (needs more work)

### CODE REVIEW

**Comment by makrsmark** [comment]
@rpatel3001 - we might want something a little extra for acars2pos so that we can put the plane on the ground or in the air. Thoughts?

File: lib/plugins/Label_13_18_Slash.ts [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+
+    decodeResult.decoded = true;
+    decodeResult.decoder.decodeLevel = decodeResult.remaining.text ? 'partial' : 'full';
+    return decodeResult;
+  }
+}
+
+export default {};
+
+function getMsgType(labelNumber : number): string {
```
Comment (rpatel3001) [inline lib/plugins/Label_13_18_Slash.ts:89]: can we somehow pull this automatically from [here](https://github.com/airframesio/data/blob/master/json/acars/metadata.json) for all messages?

**Comment by rpatel3001** [comment]
> @rpatel3001 - we might want something a little extra for acars2pos so that we can put the plane on the ground or in the air. Thoughts?

easy enough, there's a spot in the SBS format for IsOnGround, which I think tar1090 uses correctly

File: lib/MessageDecoder.ts [hunk 2]
```diff
@@ -19,7 +19,7 @@ export class MessageDecoder {
     this.registerPlugin(new Plugins.Label_10_POS(this));
     this.registerPlugin(new Plugins.Label_10_Slash(this));
     this.registerPlugin(new Plugins.Label_12_N_Space(this));
-    this.registerPlugin(new Plugins.Label_14_Slash(this));
+    this.registerPlugin(new Plugins.Label_13_18_Slash(this));
```
Comment (rpatel3001) [inline lib/MessageDecoder.ts:22]: is there a better way to name this? as is it sort of implies label 12 with preamble 18/. maybe its best to have an individual decoder for each label as a stub that calls an internal helper decoder?

Comment (makrsmark) [reply lib/MessageDecoder.ts:22]: I could do it that way - I do with H1. Trying to see if i like this way better (i think i do). I'll think about a better name and if i can't i'll go that route

**Comment by makrsmark** [comment]
understood on the SBS ground flag - do you want something exposed here for acars2pos to use? or does it have enough as-is?

**Comment by rpatel3001** [comment]
i think there's enough as is - i can check for the new OOOI values in the results object. on/in/out are on the ground and off is airborne. i don't think we can quite say that all other messages are airborne, but for sure anything with an altitude is.

**Comment by makrsmark** [comment]
Awesome! Thanks!

**Pull request merged**